jobs/client.go

```diff
@@ -0,0 +1,53 @@
+package jobs
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+)
+
+const (
+	parserError = "could not parse error"
+)
+
+type ZipClient struct {
+	BaseUrl string
+}
+
+func NewZipClient() *ZipClient {
+	apiKey := os.Getenv("ZIP_RECRUITER_KEY")
+	if len(apiKey) == 0 {
+		panic("ZIP_RECRUITER_KEY env variable not set")
+	}
+	return &ZipClient{BaseUrl: apiKey}
+}
+
+func (z *ZipClient) Get(request ZipRequest) (*ZipResponse, error) {
+	url := fmt.Sprintf("%s?search=%s&locatiion=%s&radius_miles=%s&days_ago=%s&jobs_per_page=%s&page=%s&refined_salary=%s&api_key=%s",
```

Comment:
typo `locatioon`

Reply:
please use https://golang.org/pkg/net/url/#example_URL `net/url` package is more readable.

Reply:
done